ws/shell/model.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package shell
+
+import (
+	"github.com/mendersoftware/go-lib-micro/ws"
+)
+
+var (
+	MessageTypeShellCommand = "shell"
+	MessageTypeSpawnShell   = "new"
+	MessageTypeStopShell    = "stop"
+)
+
+type MenderShellMessageStatus int
+
+const (
+	NormalMessage MenderShellMessageStatus = iota
```

Comment:
```suggestion
	NormalMessage MenderShellMessageStatus = iota + 1
```
Starting on `1` will catch errors due to uninitialized structs upon validation.